pkgs/development/python-modules/sphinx/default.nix

```diff
@@ -30,29 +30,30 @@
 
 # check phase
 , cython
+, filelock
 , html5lib
 , pytestCheckHook
 , typed-ast
 }:
 
 buildPythonPackage rec {
   pname = "sphinx";
-  version = "5.3.0";
+  version = "7.1.2";
   format = "pyproject";
 
   disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = "sphinx-doc";
-    repo = pname;
+    repo = "sphinx";
     rev = "refs/tags/v${version}";
-    hash = "sha256-80bVg1rfBebgSOKbWkzP84vpm39iLgM8lWlVD64nSsQ=";
     postFetch = ''
       cd $out
       mv tests/roots/test-images/testimäge.png \
-        tests/roots/test-images/testimæge.png
+         tests/roots/test-images/testimæge.png
```

Comment:
this space intentional?

Reply:
For alignment - since they are not in the same line, aligning them this way highlights the modified character.

(Should this be reported upstream?)

Reply:
Does anyone bother to align this way in Nixpkgs. I thought everything was always indented as 2 spaces with no regard for the tedium of alignment (tho I would prefer >2 spacing myself for readability)

Reply:
No. This was just because this patch is suspiciously specific - "some filesystems are too bugged to understand this character" or the like.
I will revert this nonetheless.